x-pack/test/functional/apps/transform/editing.ts

```diff
@@ -0,0 +1,152 @@
+
+function getTransformConfig(): TransformPivotConfig {
+  const date = Date.now();
+  return {
+    id: `ec_2_${date}`,
+    source: { index: ['ft_ecommerce'] },
+    pivot: {
+      group_by: { category: { terms: { field: 'category.keyword' } } },
+      aggregations: { 'products.base_price.avg': { avg: { field: 'products.base_price' } } },
+    },
+    description:
+      'ecommerce batch transform with avg(products.base_price) grouped by terms(category.keyword)',
+    dest: { index: `user-ec_2_${date}` },
+  };
+}
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const transform = getService('transform');
+
+  describe('editing', function () {
+    const transformConfig = getTransformConfig();
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/ecommerce');
+      await transform.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+      await transform.api.createAndRunTransform(transformConfig);
+      await transform.testResources.setKibanaTimeZoneToUTC();
+
+      await transform.securityUI.loginAsTransformPowerUser();
+    });
+
+    after(async () => {
+      await transform.testResources.deleteIndexPatternByTitle(transformConfig.dest.index);
+      await transform.api.deleteIndices(transformConfig.dest.index);
+      await transform.api.cleanTransformIndices();
+    });
+
+    const testData = {
+      suiteTitle: 'edit transform',
+      transformDescription: 'updated description',
+      transformDocsPerSecond: '1000',
+      transformFrequency: '10m',
+      expected: {
+        messageText: 'updated transform.',
+        row: {
+          status: 'stopped',
+          mode: 'batch',
+          progress: '100',
+        },
+      },
+    };
+
+    describe(`${testData.suiteTitle}`, function () {
+      after(async () => {
+        // await transform.api.deleteIndices(<CLONE_DEST_INDEX>);
+      });
+
+      it('should load the home page', async () => {
+        await transform.navigation.navigateTo();
+        await transform.management.assertTransformListPageExists();
+      });
+
+      it('should display the transforms table', async () => {
+        await transform.management.assertTransformsTableExists();
+      });
+
+      it('should display the original transform in the transform list', async () => {
+        await transform.table.refreshTransformList();
+        await transform.table.filterWithSearchString(transformConfig.id);
+        const rows = await transform.table.parseTransformTable();
+        expect(rows.filter((row) => row.id === transformConfig.id)).to.have.length(1);
+      });
+
+      it('should show the actions popover', async () => {
+        await transform.table.assertTransformRowActions(false);
+      });
+
+      it('should show the edit flyout', async () => {
+        await transform.editFlyout.openTransformEditFlyout();
```

Comment:
You're doing it in a slightly< different way in the cloning test:
```
          await transform.table.clickTransformRowAction('Clone');
          await transform.wizard.assertDefineStepActive();
```
Even though the `openTransformEditFlyout` is working fine, I'd suggest to do things in a similar way. Personally, I prefer the cloning version.

Reply:
Updated in a7d3a20.